src/apps/relay/turn_admin_server.c

```diff
@@ -1557,7 +1557,8 @@ static ioa_socket_handle current_socket = NULL;
 static char *get_bold_admin_title(void)
 {
 	static char sbat[1025];
-	STRCPY(sbat,__bold_admin_title);
+	strncpy(sbat,__bold_admin_title,TURN_LONG_STRING_SIZE);
```

Comment:
sizeof(sbat) looks safer

or alternatively use TURN_LONG_STRING_SIZE also in "sbat" definition, but i think the sizeof is safer.


(This applies also to the previous file mainrelay.c)